components/esb-tools/plugins/org.wso2.integrationstudio.artifact.synapse.api/src/org/wso2/integrationstudio/artifact/synapse/api/ui/wizard/SynapseAPICreationWizard.java

```diff
@@ -147,6 +161,12 @@ public class SynapseAPICreationWizard extends AbstractWSO2ProjectCreationWizard
     private static final String WHITE_SPACE = " ";
     private static final String METADATA_TYPE = "synapse/metadata";
     private static final String REGISTRY_RESOURCE_PATH = "/_system/governance/swagger_files";
+    private static final String SWAGGER_CONVERTER_API_URL = "https://converter.swagger.io/api/convert";
+    private static final String SWAGGER_API_CALL_CONSENT_TITLE = "Convert to Swagger version 3?";
+    private static final String SWAGGER_API_CALL_CONSENT_MESSAGE = "Integration Studio supports Swagger version 3 and above only. The Swagger file you have uploaded is version 2. You can convert this file to version 3 by clicking “Yes”. Note that Integration Studio uses the https://converter.swagger.io API to convert your payload.\n\nYou can continue with the Swagger file you uploaded by clicking “No”, if you feel the detected version is incorrect.\n\nClick “Cancel” to terminate the API creation process.";
```

Comment:
Break the line at 120 length

Reply:
This PR is reverted by: https://github.com/Tharsanan1/integration-studio/commit/ef1e82bf7abe20bfb8300d844d2533ca4ffbb6dd. Hence resolving the comment